Incorrect argument type passed to IUserManager.GetUserById

```
Api/ReportsService.cs(111,94): error CS1503: Argument 1: cannot convert from 'string' to 'System.Guid' [/home/oddstr13/Projects/Jellyfin/Plugins/jellyfin-plugin-reports/Jellyfin.Plugin.Reports/Jellyfin.Plugin.Reports.csproj]
Api/ReportsService.cs(145,94): error CS1503: Argument 1: cannot convert from 'string' to 'System.Guid' [/home/oddstr13/Projects/Jellyfin/Plugins/jellyfin-plugin-reports/Jellyfin.Plugin.Reports/Jellyfin.Plugin.Reports.csproj]
```

`IUserManager.GetUserById` now takes `Guid` and not `string`.

Wrapping `request.UserId` in `new Guid()` makes the error go away, but I'm not sure if this is the correct way to do this.

https://github.com/jellyfin/jellyfin-plugin-reports/blob/8ffc8ba47d2f8c4a72cdd03322750e2daa46f868/Jellyfin.Plugin.Reports/Api/ReportsService.cs#L111
https://github.com/jellyfin/jellyfin-plugin-reports/blob/8ffc8ba47d2f8c4a72cdd03322750e2daa46f868/Jellyfin.Plugin.Reports/Api/ReportsService.cs#L145

--- a/Jellyfin.Plugin.Reports/Api/ReportsService.cs
+++ b/Jellyfin.Plugin.Reports/Api/ReportsService.cs
@@ -108,7 +108,7 @@ namespace Jellyfin.Plugin.Reports.Api
                 return null;
 
             request.DisplayType = "Screen";
-            var user = !string.IsNullOrWhiteSpace(request.UserId) ? _userManager.GetUserById(request.UserId) : null;
+            var user = !string.IsNullOrWhiteSpace(request.UserId) ? _userManager.GetUserById(new Guid(request.UserId)) : null;
             var reportResult = GetReportResult(request, user);
 
             return reportResult;
@@ -142,7 +142,7 @@ namespace Jellyfin.Plugin.Reports.Api
             headers["Content-Disposition"] = string.Format("attachment; filename=\"{0}\"", filename);
             headers["Content-Encoding"] = "UTF-8";
 
-            var user = !string.IsNullOrWhiteSpace(request.UserId) ? _userManager.GetUserById(request.UserId) : null;
+            var user = !string.IsNullOrWhiteSpace(request.UserId) ? _userManager.GetUserById(new Guid(request.UserId)) : null;
             ReportResult result = null;
             switch (reportViewType)
             {

